config/initializers/hesa_disabilities.rb

```diff
@@ -1,12 +1,26 @@
+module HesaDisabilityTypes
```

Comment:
I'm in two minds about this. The use of constants has been used in order to have 'one source of truth'. This is useful in scenarios where a form value does not directly map onto a HESA value. On the other hand I'm worried this creates more indirection and makes the code less easier to understand. Any opinions on this welcome!